packages/dds/tree/src/test/shared-tree/summaryBenchmarkSizes.ts

```diff
@@ -0,0 +1,221 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { TransactionResult } from "../../checkout";
+import { FieldKinds, PlacePath, singleTextCursor } from "../../feature-libraries";
+import { ISharedTree } from "../../shared-tree";
+import { detachedFieldAsKey, FieldKey, rootFieldKey, TreeValue } from "../../tree";
+import { brand } from "../../util";
+import { TestTreeProvider } from "../utils";
+import { fieldSchema, GlobalFieldKey, namedTreeSchema, SchemaData } from "../../schema-stored";
+
+const globalFieldKey: GlobalFieldKey = brand("globalFieldKey");
+
+describe("Summary size benchmark", () => {
+    it("for different sized trees", async () => {
+        const output = await getSummaryBenchmarkSizes();
+        assert(output.summarySizeWideInsert1 < 1000);
+        assert(output.summarySizeWideInsert10 < 3000);
+        assert(output.summarySizeWideInsert100 < 20000);
+        assert(output.summarySizeNarrowInsert10 < 3000);
+        assert(output.summarySizeNarrowInsert100 < 20000);
+        assert(output.summarySizeNoEdits < 1000);
+    });
+    it("for 1000 inserts width wise", async () => {
+        await assert.rejects(getInsertsSummarySize(1000, 0, true), { message: "BatchTooLarge" });
+    });
+});
+
+export interface summarySizeBenchmarks {
```

Comment:
If you keep this type, capitalize it (we capitalize all type names).

Reply:
Removed interface